pom.xml

```diff
@@ -458,6 +458,13 @@
         <artifactId>javax.servlet-api</artifactId>
         <version>${javax-servlet-api.version}</version>
       </dependency>
+
+      <dependency>
+        <groupId>org.apache.jspwiki</groupId>
+        <artifactId>jspwiki-util</artifactId>

```

Comment:
this shouldn't be needed here, jspwiki modules should be referenced on other modules using `groupId:artifactId:version` where `groupId` is always `${project.groupId}` and version always is `${project.version}` (see [this](https://github.com/apache/jspwiki/blob/master/jspwiki-main/pom.xml#L34-L36) for an example)